pkg/bytesutil/bytesutil.go

```diff
@@ -155,7 +152,7 @@ func CloneSlice(a [][]byte) [][]byte {
 // Pack converts a sparse array to a dense one.  It removes sections of a containing
 // runs of val of length width.  The returned value is a subslice of a.
 func Pack(a []byte, width int, val byte) []byte {
-	var i, j, iStart, jStart, end int
+	var i, j, jStart, end int
```

Comment:
@jwilder  `iStart` was completely unused. Was it supposed to be used? i.e., was/is this code buggy?